src/GitUtils.ps1

```diff
@@ -569,6 +635,20 @@ function Add-SshKey([switch]$Quiet) {
 
 # Stop a running SSH agent
 function Stop-SshAgent() {
+    $nativeAgent = Get-NativeSshAgent
+
+    if ($nativeAgent) {
+        if ($nativeAgent.Status -eq 'Running') {
+            if (Test-Administrator) {
+                Stop-Service $nativeAgent
+            }
+            else {
+                write-host Access denied. Please open a prompt as administrator and try again.
```

Comment:
`Write-Error` (with quoted message) here too, I think.

Reply:
Parts of this (Stop-Service) are Windows specific and should be firewalled off on Linux/macOS.

Reply:
- Switched to Write-Error
- `Get-NativeSshAgent` now does an OS check, so an explicit check in `Stop-Service` shouldn't be necessary